### PR TITLE
Add Serverless and IOT Pune Summit

### LINKED ISSUE #36: Lengthy conference descriptions
@nikhita peoples again commit with some large descriptions :(

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #36.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ If you would like to be notified of future conferences that would be added to th
 | [Grace Hopper Celebration India (GHCI)](https://ghcindia.anitaborg.org/) | 16-17 Nov |  Bengaluru International Exhibition Center (BIEC), Bengaluru | The Grace Hopper Celebration of Women in Computing (India) is India's largest gathering of women technologists. |
 | [DataHack Summit](https://www.analyticsvidhya.com/datahacksummit/) | 9-11 Nov | Bengaluru | DataHack Summit is an initiative by Analytics Vidhya to bring together the finest data scientists from across the country & globe. |
 | [PyCon India](https://in.pycon.org/2017/) | 2-5 Nov | Jawahar Lal Nehru University, New Delhi | The premier conference in India on using and developing the Python programming language. |
+| [Serverless Summit](http://inserverless.com/) | 27 October | Bangalore | India’s first confluence on serverless technologies. |
```
Comment (nikhita) [inline README.md]: Can you rename `October` to just `Oct`?

File: README.md [hunk 2]
```diff
@@ -34,6 +35,7 @@ If you would like to be notified of future conferences that would be added to th
 | [DevOps Summit](http://www.unicomlearning.com/2017/DevOps_Summit_Pune/) | 16 Sept | Pune | The conference brings together leading practitioner-organisations who have achieved Continuous Delivery and organisational transformation. |
 | [JSFoo](https://jsfoo.in/2017/) | 15-16 Sept | MLR Convention Centre, Bengaluru | JSFoo is India’s premier JavaScript conference, hosted by HasGeek. The broad theme this year is building reliable web apps.  |
 | [DevOps Days India](https://devopsdaysindia.org/) | 15-16 Sept | Hotel Vivanta by Taj, MG Road, Bengaluru | DevOpsDays India is a global event complementing other DevOpsDays events across the world. |
+| [Iot Summit](http://www.unicomlearning.com/2017/IoT_Summit_Pune/)| 15 September | Pune | The conference explores how the Internet of Things will re-shape our interactions with the real and virtual worlds in the coming year.|
```
Comment (nikhita) [inline README.md]: Same here. `September` to `Sept`. I guess this error occurs for some other fields as well. Can you change for them too?

Comment (championshuttler) [inline README.md]: ok doing for all summits wait for next PR , close this for now :D

Comment (nikhita) [inline README.md]: @championshuttler please don't close this. Make the changes in this PR.

**Comment by championshuttler** [comment]
now you can merge it i think :)

File: README.md [hunk 3]
```diff
@@ -11,54 +11,56 @@ If you would like to be notified of future conferences that would be added to th
...
-| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google and many contributors from the open source community. |
+| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google. |
 | [PyCon Pune](https://pune.pycon.org/2018/) | 08-11 Feb | Pune | The community will meet for two days of talks and work on upstream projects in two days of dev sprint. |
-| [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on latest happenings in the field of web technologies. Hosting, Domain, Blogging and Web technologies are major discussions. |
+| [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on latest happenings in the field of web technologies. |
```
Comment (nikhita) [inline README.md]: Can you update this to: `intiative to discuss hosting, domain, blogging and web technologies`?

File: README.md [hunk 4]
```diff
@@ -11,54 +11,56 @@ If you would like to be notified of future conferences that would be added to th
...
 |------------|------|-------|-------------|
 | [World Cloud Summit](http://worldcloudsummit.com/) | 23-24 Nov | Jaipur | World Cloud Summit is one of the largest gatherings of cloud services and internet infrastructure industries. |
-| [XP Conference India 2017](http://xpconference.in/2017XpConf/) | 17-18 Nov | Bengaluru | The 3rd International XP Conference on Agile Engineering practices for software product development, hosted by Volunteers and the community at large. |
+| [XP Conference India 2017](http://xpconference.in/2017XpConf/) | 17-18 Nov | Bengaluru | The 3rd International XP Conference on Agile Engineering practices for software product development. |
 | [Functional Conf](https://functionalconf.com/) | 16-19 Nov | Bengaluru | Asia's Premier Functional Programming Conference, designed to bring the growing community of functional programmers together under one roof. |
-| [try! swift](https://www.tryswift.co/events/2017/bangalore/) | 16-17 Nov | MLR Convention Centre, Bengaluru | try! Swift India is an amazing chance for developers in the Asian Pacific region to learn the latest world trends in iOS, tvOS, and watchOS development using the industry's best standards. |
```
Comment (nikhita) [inline README.md]: Let's keep this. The main topic is about iOS, etc in this case.

File: README.md [hunk 5]
```diff
@@ -11,54 +11,56 @@ If you would like to be notified of future conferences that would be added to th
...
+| [CYPHER 2017](http://www.analyticsindiasummit.com/about/) | 21-23 Sept | Hotel Park Plaza, Bengaluru | A platform to network and learn Analytics, Data Science and AI discipline. |
 | [DevOps Summit](http://www.unicomlearning.com/2017/DevOps_Summit_Pune/) | 16 Sept | Pune | The conference brings together leading practitioner-organisations who have achieved Continuous Delivery and organisational transformation. |
 | [JSFoo](https://jsfoo.in/2017/) | 15-16 Sept | MLR Convention Centre, Bengaluru | JSFoo is India’s premier JavaScript conference, hosted by HasGeek. The broad theme this year is building reliable web apps.  |
 | [DevOps Days India](https://devopsdaysindia.org/) | 15-16 Sept | Hotel Vivanta by Taj, MG Road, Bengaluru | DevOpsDays India is a global event complementing other DevOpsDays events across the world. |
+| [Iot Summit](http://www.unicomlearning.com/2017/IoT_Summit_Pune/)| 15 Sept | Pune | The conference explores how the Internet of Things will re-shape our interactions with the real and virtual worlds in the coming year.|
```
Comment (nikhita) [inline README.md]: `Iot` => `IOT`

File: README.md [hunk 6]
```diff
@@ -11,54 +11,56 @@ If you would like to be notified of future conferences that would be added to th
...
+|[Data Visualization Summit](http://www.unicomlearning.com/2017/Data_Visualization_Summit_Mumbai/)| 1 Sept| Mumbai | A primary goal of data visualization is to communicate information clearly and efficiently via statistical graphics. |
+| [Oracle Code Bengaluru ](https://developer.oracle.com/code/bangalore) | 10 Aug | Bengaluru, Karnataka | Oracle Code is a free event for developers to learn about the latest developer technologies, practices, and trends.|
+| [JS Channel](http://2017.jschannel.com/) | 28-29 July | JW Marriott Hotel, Bengaluru | JS Channel is a unique platform to hear bests speakers in JS Community. |
+| [Eclipse Summit India](https://eclipsesummit.in/) | 29 July | Infosys Campus,Bengaluru | Eclipse Summit India is the 2nd annual conference presented by the local Eclipse Community |
```
Comment (nikhita) [inline README.md]: :+1:

File: README.md [hunk 7]
```diff
@@ -11,54 +11,56 @@ If you would like to be notified of future conferences that would be added to th
...
 | [Anthill Inside](https://anthillinside.in/2017/) | 29 July | MLR Convention Centre, Whitefield, Bengaluru | Anthill Inside is the new avataar of the Deep Learning conference.|
-| [The Fifth Elephant](https://fifthelephant.in/2017/) | 27-28 July | MLR Convention Centre, Whitefield, Bengaluru | The Fifth Elephant is India’s most renowned conference on data, a space for discussing the most relevant tools, techniques and frameworks with expert practitioners in the field. |
+| [The Fifth Elephant](https://fifthelephant.in/2017/) | 27-28 July | MLR Convention Centre, Bengaluru | The Fifth Elephant is India’s most renowned conference on data. |
 | [SANOG](http://www.sanog.org/sanog30/) | 10-18 July | Ramada Gurgaon Central, Gurgaon, Haryana, India | The South Asian Network Operator Group. Engineers sharing operational knowledge on running ISPs and modern networks. |
-| [AlterConf](https://alterconf.com/conferences/bangalore-india) (postponed to 2018) | 8 July | Microsoft Signature Building, Bengaluru | AlterConf is a traveling conference series that provides safe opportunities for marginalized people and those who support them in the tech and gaming industries. |
```
Comment (nikhita) [inline README.md]: Let's keep this too. tech and gaming are two main topics for AlterConf.

File: README.md [hunk 8]
```diff
@@ -76,15 +78,15 @@ If you would like to be notified of future conferences that would be added to th
...
-| [IndiaHacks Conference](https://www.hackerearth.com/indiahacks-conference/) | 19 Mar | Vivanta by Taj, Bengaluru | Conducted by HackerEarth,the conference has an extensive lineup of speakers talking on a variety of technical subjects like Machine Learning, Big Data, Artificial Intelligence, Scientific Computing, etc. |
+| [SANOG](http://www.sanog.org/sanog28/) | 1-9 Aug | Trident, BKC, Mumbai | The South Asian Network Operator Group. Engineers sharing operational knowledge on running ISPs and modern networks. |
+| [The Fifth Elephant](https://fifthelephant.in/2016/) | 28-29 July | NIMHANS Convention Centre, Bengaluru | The Fifth Elephant is India’s most renowned conference on data. |
+| [Rootconf](https://rootconf.in/2016/) | 14-15 Apr | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge. |
+| [IndiaHacks Conference](https://www.hackerearth.com/indiahacks-conference/) | 19 Mar | Vivanta by Taj, Bengaluru | Conference has an extensive lineup of speakers talking on a variety of technical subjects like ML, Big Data, AI etc. |
```
Comment (nikhita) [inline README.md]: Instead of `Conference has`, please make it `IndiaHacks has`. Btw, nice idea about shortening it to ML and AI.

**Comment by nikhita** [comment]
@championshuttler Also, please squash the commits to 2 commits. :smile: 

- one for adding IOT Summit and Serverless Summit
- one for shortening the descriptions and month names.

File: README.md (hunk 3 above)
Comment (championshuttler) [inline README.md]: ok :)

File: README.md (hunk 5 above)
Comment (championshuttler) [inline README.md]: ok :)

File: README.md (hunk 6 above)
Comment (championshuttler) [inline README.md]: :)

File: README.md (hunk 7 above)
Comment (championshuttler) [inline README.md]: Sure

File: README.md (hunk 8 above)
Comment (championshuttler) [inline README.md]: ok :)

**Comment by championshuttler** [comment]
i dont know how to do squash commits , will try to do if not then can you do it by yourself? @nikhita

**Comment by nikhita** [comment]
@championshuttler Looks good now! :+1: 

I can't selectively squash from the Github UI so you will have to squash it yourself. Let me know if you need any help with that.

**Pull request merged**